apps/cns-website/src/app/pages/people-profile/components/contact-info/contact-info.component.scss

```diff
@@ -0,0 +1,65 @@
+@use 'vars' as vars;
+
+:host {
+  display: flex;
+  flex-direction: column;
+  min-height: 100%;
+
+  @media (max-width: 639px) {
```

Comment:
Always design mobile first, i.e. mobile styles should be the default and desktop specific styles should be applied with media queries